lib/ansible/modules/network/avi/avi_serviceenginegroup.py

```diff
@@ -85,13 +91,14 @@
         type: bool
     app_cache_percent:
         description:
-            - A percent value of total se memory reserved for application caching.
-            - This is an se bootup property and requires se restart.
+            - A percent value of total se memory reserved for applicationcaching.
+            - This is an se bootup property and requires se restart.requires se reboot.
```

Comment:
I'm confused here. Does it require se restart, or se reboot, or both? Or are they the same thing?